eZ/Publish/Core/Persistence/Legacy/Content/Mapper.php

```diff
@@ -430,10 +432,11 @@ protected function extractFieldValueFromRow(array $row, $type)
      * Creates CreateStruct from $content.
      *
      * @param \eZ\Publish\SPI\Persistence\Content $content
+     * @param bool $preserveMainLanguage
      *
      * @return \eZ\Publish\SPI\Persistence\Content\CreateStruct
      */
-    public function createCreateStructFromContent(Content $content)
+    public function createCreateStructFromContent(Content $content, $preserveMainLanguage = false)
```

Comment:
does not need the argument here, it can always set the main language code as we here always make it using a content that has it.